book-service/src/main/resources/application.yml

```diff
@@ -1,15 +1,31 @@
 # GLOBAL CONFIGURATION
 
 spring:
+  cloud:
+    config:
+      uri: http://localhost:8888
+      enabled: false
   application.name: book-service
```

Comment:
`application.name` not sure if it's working. Don't mix yml notation with properties file notation.